src/Window.res

```diff
@@ -118,6 +118,9 @@ external userAgent: string = "userAgent"
 @val @scope("navigator")
 external sendBeacon: (string, string) => unit = "sendBeacon"
 
+@val @scope("navigator")
+external language: string = "language"
```

Comment:
I'll recommend moving all `navigator` scoped variables under `module Navigator = {}` in this file, so that if someone needs to use such variable, they would need to do `Window.Navigator.language` instead of `Window.language`.

It is better if the rescript code is as close to corresponding JS code as possible. Corresponding JS code for this is `window.navigator.language` and not `window.language`. So I'll recommend moving these declarations into a sub-module

Reply:
pritish already informed me about this, I will raise a separate pr for the same